go/ai/option.go

```diff
@@ -430,6 +446,20 @@ func WithOutputFormat(format OutputFormat) OutputOption {
 	return &outputOptions{OutputFormat: format}
 }
 
+// WithOutputInstructions sets whether output instructions should be added.
+func WithOutputInstructions(addInstructions bool) OutputOption {
+	var instructions string
+	if addInstructions {
+		instructions = ""
+	}
+	return &outputOptions{OutputInstructions: &instructions}
+}
+
+// WithOutputNativeConstrained sets whether to use model native constrained output generation. Defaults to true.
```

Comment:
```suggestion
// WithOutputNativeConstrained sets whether to use model native constrained output generation. If this is set to true and the model does not support native constrained generation, it will be simulated instead. Defaults to true.
```